build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -139,6 +151,14 @@ Description of event. Use cases of event-name. Include all the nuances, such as:
 
 [/example]
 
+#### Bento mode
+
+Syntax and argument details for use in Bento mode.
+
+```
+Bento example
```

Comment:
It's going to be ~the same as above, but:

1. CSS instructions as we've discussed just a few minutes ago.
2. No on-action.

Reply:
Sure, but this is for the individual events made available by the API. Take amp-accordion's `toggle` for example, we should explain that its called through `toggle()` and that when it has no arguments it toggles all accordion sections, or it accepts the id of a section as an argument to toggle a specific one. From what I've seen from provided Bento examples, there are some differences from the AMP actions and event implementations, such as not needing to include `section=` in the argument

Reply:
Correct.